utensor_cgen/ir/instr.py

```diff
@@ -0,0 +1,53 @@
+class MemoryAllocation:
```

Comment:
You should make the base class as `object` or `MemoryAllocation` is a old style class which has different typing system as new style class in python2.

Reply:
https://docs.python.org/2.5/ref/node33.html